RandLAPACK/comps/rl_util.hh

```diff
@@ -513,6 +738,42 @@ bool orthogonality_check(
     return false;
 }
 
+/// Computes an L-2 norm of a given matrix using
```

Comment:
GitHub won't let me comment on Line 725 in the ``orthogonality_check`` function, so I comment on it here.

Don't use ``gemm`` to form a gram matrix. Use ``syrk`` and then copy the data from the upper to the lower triangle (or the other way around).

Reply:
Done.